buildSrc/src/main/kotlin/org/jetbrains/exposed/gradle/Versions.kt

```diff
@@ -26,5 +26,5 @@ object Versions {
 
     /** Test Dependencies **/
     const val testContainers = "1.16.3"
-    const val otjPgEmbedded = "0.13.4"
+    const val otjPgEmbedded = "1.0.1"
```

Comment:
it was required to upgrade version, because old one doesn't work on Mac M1, meanwhile new one is based on testcontainers impl